crnk-core/src/test/resources/logback-test.xml

```diff
@@ -4,7 +4,7 @@
 	<appender name="CONSOLE" class="ch.qos.logback.core.ConsoleAppender">
 		<encoder>
 			<charset>UTF-8</charset>
-			<pattern>%d{HH:mm:ss,SSS} %-5.5p [%15.15t] [%30.30c] %X{indent}%m%n
+			<pattern>%d{HH:mm:ss} %-5.5p [%15.15t] [%30.30c] %X{indent}%m%n
```

Comment:
Try to use `%date{ISO8601}` instead of `%d{HH:mm:ss}`. Applicable to all other logger config files too.